src/order/filter/basic.lean

```diff
@@ -1676,6 +1676,26 @@ iff.intro
     (assume s₁ s₂ h ⟨a, ha⟩, ⟨a, assume b hb, h $ ha _ hb⟩))
   (assume ⟨a, h⟩, mem_infi_sets a $ assume x, h x)
 
+@[nolint] -- ≥
+lemma eventually_at_top {α} [semilattice_sup α] [nonempty α] {p : α → Prop} :
+  (∀ᶠ x in at_top, p x) ↔ (∃ a, ∀ b ≥ a, p b) :=
```

Comment:
Isn't it better from a global perspective to have the slightly more ugly statement involving `≤`?

Reply:
I used the same convention as in `mem_at_top_sets`, which uses `∀ b ≥ a`.  I thought that quite a few people were against `∀ b ≥ a,` to `∀ b, a ≤ b →`, so I didn't attempt to do that.

Reply:
Ok, I wasn't sure if there was some convention or not. Anyway, it's fine with me. We can always regexp them into the other form if we need that.